packages/core/types/index.js

```diff
@@ -1979,7 +1979,8 @@ export type ValidationEvent = {|
  * @section reporter
  */
 export type TraceEvent = {|
-  +type: 'trace',
+  +type: 'trace.start' | 'trace.end',
+  +id: string,
   +ts: number,
   +duration: number,
```

Comment:
There's a flow error in your calls to `.trace()` because the trace event expects a duration which now won't be relevant with `begin`/`end` events.